virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-images.adoc

```diff
@@ -0,0 +1,13 @@
+[id="virt-creating-and-using-default-os-images"]
+= Creating and using default OS images
+include::modules/virt-document-attributes.adoc[]
+:context: virt-creating-and-using-default-os-images
+toc::[]
+
+A _default OS image_ is a bootable disk containing an operating system and all of the operating system's configuration settings, such as drivers. You use a default OS image to create bootable virtual machines with specific configurations.
```

Comment:
"all of the operating system's configuration settings"

Suggest ->

"all of the configuration settings for the operating system"

Re: avoiding inanimate possessives (per ISG).

Reply:
Wondering if the first instance of "OS" should be "default operating system (OS) image" or not.

Reply:
Agreed. I also spelled out OS in each of the modules and assemblies at first usage then used "OS" thereafter when it wasn't spelled out in a GUI item.